### ISSUE TITLE
Some Practices Need Attribution

### ISSUE THREAD

**Issue by kattekrab** [openpracticelibrary/openpracticelibrary#1623]
There are a few practices in the library that need attribution, or credit, to the original author or practitioner.

As an example, I recently updated [T2R2](https://openpracticelibrary.com/practice/t2r2-talk-type-read-review/) to give credit to John Findlay, the creator of the meeting etiquette at the base of the practice.

**Comment by kattekrab** [comment]
ToDo:
 - create a list of all the practices, 
 - check each one to see if there is or isn't attribution
 - create an issue here, Label it content - needs attribution.

**Comment by ahussey-redhat** [comment]
I can open another issue but feel the following could tie nicely into this issue.

**Issue**: When users are reading the practices there is no reference to the author's social media handles/aliases. This means that if the user wants to start a conversation about the practice with the author, they will have to search through the different platforms for the author. Secondary to this, when we release tweets about a practice it would be beneficial to have the author's handles readily available so we can attribute/tag the author in the tweet/post.

**Suggested resolution**: Add the appropriate handles after each author, in the `authors` list for each practice. IE;
```yaml
authors:
  - author1 @author1
  - author2 @author2
```

**Comment by ahussey-redhat** [comment]
https://github.com/openpracticelibrary/openpracticelibrary/issues/1574

**Comment by kattekrab** [comment]
Yes!  Thanks @ahussey-redhat - agreed - and that came up during some UX research interviews I ran last year.
Two people expressed their expectation of being able to click on the contributor and see some kind of profile.

**Comment by kattekrab** [comment]
I forgot about this approach too...  seems there's a few ways to tackle this.
What's best? / easiest?

**Comment by ahussey-redhat** [comment]
@kattekrab , just for completeness I'll add the comment in here that captures our descision.

>update: @ahussey-redhat  and I ( @kattekrab ) had a chat today about how to 
  handle twitter IDs in OPL, and agreed to go ahead with adding a field to the 
  author field - I've made a start of trying to collect the twitter handles - so we can 
  then backfill the data once we have the field

**Comment by kattekrab** [comment]
We went off in a different direction for this issue, and resolved that elsewhere.

So I'd like to come back to this and do a content review blitz. 

Most of the practices in the library are proven, tried, and true, and originated somewhere, by someone long ago.  

I would love to ensure that our "What" section mentions the original author, or creator of the practice wherever possible.

For instance, 1-2-4-all is a liberating structure.  We link to LS at the bottom, but we don't make that clear at the start.

Another example - 6 hats - was created by Edward de Bono - and that is clear.

T2R2 - was inspired by John Findlay's Anyzing / Grouputer meeting etiquette.

@eliseelk - is this something you'd like to tackle?

**Comment by eliseelk** [comment]
I'd love to help find the solution/s to this, I am starting on a project tomorrow so I can't commit a lot of time right now, what timeline were you thinking of for this @kattekrab ? Perhaps a meeting would be best to discuss pathways forward?

**Comment by kattekrab** [comment]
Thanks @eliseelk - definitely not urgent, and no specific timeline in mind, I was just reviewing the issue queue and this popped out as something I wanted to revisit!

**Comment by kattekrab** [comment]
A thought: Maybe to break it down into manageable chunks, we could start by reviewing the top 20 practices according to analytics, and do 4 or 5 each, log the results somewhere, maybe even here in github somehow.

But am open to suggesion on this, I hadn't really thought much about how to tackle it! Yes, a meeting is probably a good idea,

**Comment by eliseelkerton** [comment]
maybe we can wrap issue #485 into this, adding in photo credits

**Comment by rmarting** [comment]
IMHO this issue can be closed as in the current status of the catalog there is no practice without an author.

This command shows that all the practices has the `authors` tag:

```shell
on 🎩 ❯ grep -L 'authors:' src/pages/practice/*.md | wc -l
0
```

Is there another evidence of a practice without a proper author?

**Comment by eliseelkerton** [comment]
I'm unsure if there is evidence of other practices without a proper author, but happy for it to be closed if the command returns that result.
@kattekrab thoughts?
